src/components/drawer/drawer.scss

```diff
@@ -0,0 +1,23 @@
+@import "@material/drawer/mdc-drawer";
+@import "@material/list/mdc-list";
+@import "@material/button/mdc-button";
+
+.drawer {
+  height: 100%;
+  &__list-item {
+    position: relative;
+    &-more {
```

Comment:
please do not go deeper with naming like here

```scss
.drawer {
  ...
  &__list-item {
     ...
     &-more {
        ....
        &-icon {
             ...
        }
     }
  }
}
```

its too deep -- and it is uncomfortable to write so long class names -- and it also looks like creating elements of elements -- which is bad practice

instead do this

```scss
.drawer {
  ...
  &__list-item {
     ...
  }

  &__more {
        ....
   }

  &__icon {
             ...
   }
}
```